general/calendars/outlook/outlook_calendar.rst

```diff
@@ -0,0 +1,57 @@
+=====================================
+Synchronization with Outlook Calendar
+=====================================
+
+Life is a lot easier when you are able to manage all your calendars in one single place. Therefore,
+now Odoo also allows you to synchronize your Outlook Calendar with the Calendar application.
+
+To start, `Register your application with the Microsoft identity platform
+<https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app>`_ making
+sure to **follow all the steps described in the guidelines**.
+
+.. important::
+   For the **Redirect URI restrictions**, copy your Odoo server URI followed by
+   ``/microsoft_account/authentication``. Example:
+
+   .. image:: media/redirect_URIs.png
+      :align: center
+      :alt: Odoo's server URI that is accepted when microsoft returns authentication
+
+Once you are finished, copy:
```

Comment:
```suggestion
Configuration on your Odoo database
-----------------------------------

Once you are finished, copy:
```

I suggest you start by explaining how to activate the feature, then how to find the relevant keys in Microsoft Azure.